modules/ROOT/pages/async-scope-reference.adoc

```diff
@@ -4,48 +4,49 @@ include::_attributes.adoc[]
+Async is useful for executing time-consuming operations that do not require a response to the initiating flow, for example, when printing a file or connecting to an email server.
 
-To facilitate simultaneous branch processing, the Async scope sends one copy of the message it has received to the first embedded message processor in its own processing block. At the same time, it sends another copy of the message to the next message processor in the main flow.
+To facilitate simultaneous branch processing, the Async scope sends one copy of the Mule event (the Mule message and any Mule variables) it receives to the first processor within its processing block. At the same time, the scope sends another copy of this event to the next processor in the main flow.
 
 image::async-scope-schematic.png[Async+scope+schematic]
 
-Because the Async scope is executed in a "fire and forget" manner, the result of the processing within the scope is not available in the main flow.
+Because the Async scope executes in a "fire and forget" manner, the result of the processing within the scope is not available in the main flow. Mule variables created within the flow do not propagate to the main flow, nor do any changes to the payload or other parts of the Mule event. 
 
-== Async Configuration
+== Reference
 
 Async scopes are configurable.
 
-[%header,cols="1,4"]
+[%header,cols="1a,1a,1a,4a"]
 |===
-| Field | Description
-| Display Name (`name`) | Name for the Async scope.
-| Max Concurrency (`maxConcurrency`) a| Optional. Sets the maximum number of concurrent messages that the scope can process. By default, the container thread pool determines the maximum number of threads to use to optimize the performance when processing messages. When the scope is processing the maximum number of concurrent messages, it cannot receive additional requests.
+| Field | XML | Default | Description
+| *Display Name* | `name` | Async | Name for the Async scope.
+| *Max Concurrency* | `maxConcurrency` | See description. |  Optional. Sets the maximum number of concurrent Mule messages or variables that the scope can process. By default, the container thread pool optimizes performance by determinng the maximum number of threads to use. 
 
-Set `maxConcurrency` to `1` to cause the scope to process requests one at a time.
+Setting `maxConcurrency` to `1` causes the scope to process requests one at a time.
 
-See xref:execution-engine.adoc#backpressure[Back-Pressure Management] for details about Mule behavior after reaching the maximum concurrency value.
+Upon reaching maximum concurrency,the scope cannot receive additional requests. See xref:execution-engine.adoc#backpressure[Back-Pressure Management] for details about the behavior of Mule when it reaches the maximum concurrency value.
 |===
 
 == Async Scopes versus Subflows
```

Comment:
TODO: Remove this section and just talk about async characteristics, not in relation to Subflows

Reply:
TODO: @IvanAndresFritzler check if any scope inherits exception strategy from main scope